include/mata/nfa-strings.hh

```diff
@@ -119,6 +119,14 @@ namespace Strings {
      */
     std::set<std::pair<int, int>> get_word_lengths(const Nfa::Nfa& aut);
 
+    /**
+     * @brief Checks if the automaton @p nfa accepts only a single word \eps. 
+     * 
+     * @param nfa Input automaton
+     * @return true iff L(nfa) = {\eps}
+     */
+    bool is_lang_eps(const Nfa::Nfa& nfa);
+
```

Comment:
I would put this function beside `Mata::Nfa::is_lang_empty()` in `nfa.hh`. This function is general enough to be in the default interface.

Reply:
I don't think so. To check whether the language is epsilon doesn't seem to me as a general purpose function. It is a similar discussion as for the function computing shortest strings.

Reply:
Thinking it over again, I agree with you that the function should be kept in strings-specific interface. Closing this conversation now.